L1Trigger/L1TTrackerDTC/interface/Settings.h

```diff
@@ -0,0 +1,332 @@
+    const int offsetDetIdDSV_;      // tk layout det id minus DetSetVec->detId
+    const int offsetDetIdTP_;       // tk layout det id minus TrackerTopology lower det id
+
+    // router parameter
+    const double enableTruncation_;  // enables emulation of truncation
+    const double freqDTC_;           // Frequency in MHz, has to be integer multiple of FreqLHC
+    const int tmpTFP_;               // time multiplexed period of track finding processor
+    const int numFramesInfra_;       // needed gap between events of emp-infrastructure firmware
+    const int numRoutingBlocks_;     // number of systiloic arrays in stub router firmware
+    const int sizeStack_;            // fifo depth in stub router firmware
+
+    // converter parameter
+    const int widthRowLUT_;   // number of row bits used in look up table
+    const int widthQoverPt_;  // number of bits used for stub qOverPt
+
+    // Tracker parameter
+    const int numRegions_;             // number of phi slices the outer tracker readout is organized in
+    const int numOverlappingRegions_;  // number of regions a reconstructable particles may cross
+    const int numDTCsPerRegion_;       // number of DTC boards used to readout a detector region
+    const int numModulesPerDTC_;       // max number of sensor modules connected to one DTC board
+    const int tmpFE_;                  // number of events collected in front-end
+    const int widthBend_;              // number of bits used for internal stub bend
+    const int widthCol_;               // number of bits used for internal stub column
+    const int widthRow_;               // number of bits used for internal stub row
+    const double baseBend_;            // precision of internal stub bend in pitch units
+    const double baseCol_;             // precision of internal stub column in pitch units
+    const double baseRow_;             // precision of internal stub row in pitch units
+    const double bendCut_;             // used stub bend uncertainty in pitch units
+    const double freqLHC_;             // LHC bunch crossing rate in MHz
+
+    // f/w constants
+    const double speedOfLight_;  // in e8 m/s
+    const double bField_;        // in T
+    const double bFieldError_;   // accepted difference to EventSetup in T
+    const double outerRadius_;   // outer radius of outer tracker in cm
+    const double innerRadius_;   // inner radius of outer tracker in cm
+    const double maxPitch_;      // max strip/pixel pitch of outer tracker sensors in cm
+
+    // format specific parameter
+    const double maxEta_;        // cut on stub eta
+    const double minPt_;         // cut on stub pt, also defines region overlap shape in GeV
+    const double chosenRofPhi_;  // critical radius defining region overlap shape in cm
+    const int
+        numLayers_;  // max number of detector layer connected to one DTC (hybrid) / number of detector layers a reconstructbale particle may cross (tmtt)
```

Comment:
better to put long comment on a separate to avoid awkward code format

Reply:
done